.github/workflows/pcw.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+name: public cloud watcher
+'on': push
```

Comment:
should this be `'on'` instead of `on`?

Reply:
Both is supported, just with `'on'` yamllint doesn't complain about a boolean statement should be `true` or `false`. But I'll have another check!

Reply:
This one:

    .github/workflows/pcw.yaml
      4:1       warning  truthy value should be one of [false, true]  (truthy)

Reply:
Still doesn't want to :disappointed: